### PR TITLE
Added place_id field and language parameter

### PR DESCRIPTION
**Pull request by chester89**
For #11

### CODE REVIEW

File: src/GeocodeSharp/Google/GeocodeClient.cs [hunk 1]
```diff
@@ -52,19 +52,20 @@ public GeocodeClient(string clientId, string cryptoKey)
...
+        /// <param name="region">The region code, specified as a ccTLD ("top-level domain") two-character value. This parameter will only influence, not fully restrict, results from the geocoder.</param>
+        /// <param name="language"> The language in which to return results. Address components will all be returned in the same language, which is chosen from the first component. Should names not be available in the preferred language, the closest match will be used.</param>
+        /// <param name="filter">A component filter for which you wish to obtain a geocode. The component filter swill fully restrict the results from the geocoder. Only the results that match all the filters will be returned. Each address component can only be specified either in the address parameter or as a component filter, but not both. Doing so may result in ZERO_RESULTS.</param>
+        /// <returns>The geocode response.</returns>
+        public async Task<GeocodeResponse> GeocodeAddress(string address, string region = null, string language = null, ComponentFilter filter = null)
```
Comment (bcuff) [inline src/GeocodeSharp/Google/GeocodeClient.cs]: Can you put the new `language` parameter at the end? That way we won't break any existing code.

File: src/GeocodeSharp/Google/IGeocodeClient.cs [hunk 2]
```diff
@@ -10,10 +10,11 @@ public interface IGeocodeClient
...
+        /// <param name="region">The region code, specified as a ccTLD ("top-level domain") two-character value. This parameter will only influence, not fully restrict, results from the geocoder.</param>
+        /// /// <param name="language"> The language in which to return results. Address components will all be returned in the same language, which is chosen from the first component. Should names not be available in the preferred language, the closest match will be used.</param>
         /// <param name="filter">A component filter for which you wish to obtain a geocode. The component filter swill fully restrict the results from the geocoder. Only the results that match all the filters will be returned. Each address component can only be specified either in the address parameter or as a component filter, but not both. Doing so may result in ZERO_RESULTS.</param>
         /// <returns>The geocode response.</returns>
-        Task<GeocodeResponse> GeocodeAddress(string address, string region = null, ComponentFilter filter = null);
+        Task<GeocodeResponse> GeocodeAddress(string address, string region = null, string language = null, ComponentFilter filter = null);
```
Comment (bcuff) [inline src/GeocodeSharp/Google/IGeocodeClient.cs]: same here.

Comment (chester89) [inline src/GeocodeSharp/Google/IGeocodeClient.cs]: Sure

2016-01-26 23:59 GMT+03:00 Brandon <notifications@github.com>:

> In src/GeocodeSharp/Google/IGeocodeClient.cs
> <https://github.com/bcuff/GeocodeSharp/pull/12#discussion_r50899020>:
>
> >          /// <param name="filter">A component filter for which you wish to obtain a geocode. The component filter swill fully restrict the results from the geocoder. Only the results that match all the filters will be returned. Each address component can only be specified either in the address parameter or as a component filter, but not both. Doing so may result in ZERO_RESULTS.</param>
> >          /// <returns>The geocode response.</returns>
> > -        Task<GeocodeResponse> GeocodeAddress(string address, string region = null, ComponentFilter filter = null);
> > +        Task<GeocodeResponse> GeocodeAddress(string address, string region = null, string language = null, ComponentFilter filter = null);
>
> same here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bcuff/GeocodeSharp/pull/12/files#r50899020>.
>



-- 
Yours faithfully,
Gleb

**Comment by bcuff** [comment]
new package is up.

**Comment by chester89** [comment]
Great, thanks

2016-01-27 0:22 GMT+03:00 Brandon <notifications@github.com>:

> new package is up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bcuff/GeocodeSharp/pull/12#issuecomment-175235431>.
>



-- 
Yours faithfully,
Gleb

**Pull request merged**